shuffler/Dockerfile

```diff
@@ -1,7 +1,3 @@
 FROM {{ service_base_image.image }}
 
 COPY hail.jar /
-
-EXPOSE 443
```

Comment:
Do you not need the port anymore?

Reply:
This never had any effect, kubernetes ignores it.